includes/class-ajax.php

```diff
@@ -756,70 +753,64 @@ public function handle_frontend_submission() {
 
         $entry_fields = apply_filters( 'weforms_before_entry_submission', $entry_fields, $form, $form_settings, $form_fields );
 
-        $entry_id = weforms_insert_entry( [
-            'form_id' => $form_id,
-        ], $entry_fields );
-
+        $entry_id = 1;
+        $global_settings = weforms_get_settings();
+        if ( empty( $global_settings['after_submission'] ) ) {
+            $entry_id = weforms_insert_entry( [
+                'form_id' => $form_id,
+            ], $entry_fields );
         if ( is_wp_error( $entry_id ) ) {
             wp_send_json( [
                 'success' => false,
                 'error'   => $entry_id->get_error_message(),
             ] );
         }
-
-        // redirect URL
-        $show_message = false;
-        $redirect_to  = false;
-
-        if ( $form_settings['redirect_to'] == 'page' ) {
-            $redirect_to = get_permalink( $form_settings['page_id'] );
-        } elseif ( $form_settings['redirect_to'] == 'url' ) {
-            $redirect_to = $form_settings['url'];
-        } elseif ( $form_settings['redirect_to'] == 'same' ) {
-            $show_message = true;
-        } else {
-            $show_message = true;
-        }
-
-        // Fire a hook for integration
+        //Fire a hook for integration

```

Comment:
Why the change?

Reply:
This was a mahesh change. But from what I see it was to make sure the entry is created/not created depending on the settings prior to any redirects after the form is submitted.

Reply:
If this talking about the space after `//` , this has been updated with the most recent commit.